.envrc

```diff
@@ -0,0 +1,18 @@
+path="./.venv"
```

Comment:
Please use a more descriptive variable name for this variable. By convention we also write constants in `CAPS_SNAKE_CASE`.